bootbox.js

```diff
@@ -1069,7 +1069,21 @@
     if (!options.buttons) {
       options.buttons = {};
     }
-
+    //make sure backdrop is either true/false/static
+    if (!options.backdrop) {
+      if (options.backdrop === false || options.backdrop === 0) { //false values
+        options.backdrop = false;
+      } else { //null values
+        options.backdrop = 'static';
+      }
+    } else {
+      if (typeof options.backdrop === 'string' && options.backdrop.toLowerCase() === 'static') { //static string
+        options.backdrop = 'static';
+      } else {  //Anything true
+        options.backdrop = true;
+      }
+    } 
```

Comment:
```suggestion
    //make sure backdrop is either true/false/static
    if (!options.backdrop) {
      options.backdrop = 'static';
      
      if (options.backdrop === false || options.backdrop === 0) { //false values
        options.backdrop = false;
      }
    } else {
      options.backdrop = true;
      
      if (typeof options.backdrop === 'string' && options.backdrop.toLowerCase() === 'static') { //static string
        options.backdrop = 'static';
      }
    } 
```
Personally I would  try to avoid `else` statements when possible.

Reply:
@tarlepp Think you meant this?

`options.backdrop ? (options.backdrop === false || options.backdrop === 0) ? false : static;`

Mismatched parenthenses, otherwise.

Reply:
yeah, was just using my phone so most likely multiple typos :D

Reply:
Fixed. :)